src/main/java/org/gridsuite/securityanalysis/server/SecurityAnalysisController.java

```diff
@@ -96,17 +96,37 @@ public ResponseEntity<Mono<UUID>> runAndSave(@Parameter(description = "Network U
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(resultUuid);
     }
 
-    @GetMapping(value = "/results/{resultUuid}", produces = APPLICATION_JSON_VALUE)
-    @Operation(summary = "Get a security analysis result from the database")
+    @GetMapping(value = "/results/{resultUuid}/n", produces = APPLICATION_JSON_VALUE)
```

Comment:
@GetMapping(value = "/results/{resultUuid}/**n-result**"
We want the N result only

Reply:
fixed - changed other endpoints as well to match this pattern